packages/vite-plugin-cloudflare/e2e/README.md

```diff
@@ -44,9 +46,11 @@ test("can serve a Worker request", async ({ expect, seed, viteDev }) => {
 - The `waitForReady()` helper will resolve when the `vite dev` process has output its ready message, from which it will parse the url that can be fetched in the test.
 - The `fetchJson()` helper makes an Undici fetch to the url parsing the response into JSON. It will retry every 250ms for up to 10 secs to minimize flakes.
 
-## Debugging the tests
+## Debugging e2e tests
 
-You can provide the following environment variables to get access to the logs and the actual files being tested:
+You can control the logging and cleanup via environment variables:
 
-- `NODE_DEBUG=vite-plugin:test` - this will display debugging log messages as well as the streamed output from the commands being run.
-- `CLOUDFLARE_VITE_E2E_KEEP_TEMP_DIRS=1` - this will prevent the temporary directory containing the test project from being deleted, so that you can go and play with it manually.
+- Keep the temporary directory after the tests have completed: `CLOUDFLARE_VITE_E2E_KEEP_TEMP_DIRS=true`
+- See debug logs for the tests: `NODE_DEBUG=vite-plugin:test`
+- See debug logs for the mock npm registry: `NODE_DEBUG=mock-npm-registry`
+- See debug logs for Vite: `DEBUG="vite:*"`
```

Comment:
```suggestion

- See debug logs for Vite: `DEBUG="vite:*"`
```

Reply:
why add an empty line here?

Reply:
sorry my bad... I got tripped by the GitHub UI and thought you were putting empty lines between items 😓